docs/tools/idx/quickstart.md

```diff
@@ -0,0 +1,99 @@
+# Quickstart
+
+Get started exploring what's possible with IDX using the [IDX SDK](../reference/idx.md). For usage in a Command Interface, you will need to [install the IDX CLI](../reference/cli.md).
+
+!!! warning ""
+
+    :octicons-alert-16: IDX is in alpha, Libraries may be unstable and APIs are subject to change. Data created on IDX during alpha will *not* be portable to production. Please share what you're working on and report any issues in the [IDX Discord](https://chat.idx.com){:target="_blank"}.
```

Comment:
woah. Do our users building production apps on IDX know that we're not guaranteeing their profiles will persist going forward?

Reply:
This is a direct copy from our other IDX page. I'm not certain on if this warning is more of a Ceramic TestNet warning or if it's an IDX problem. @PaulLeCam could you weigh in?

Reply:
We've had this messaging in the docs for as long as I can remember, mostly warning that until IDX reaches a v1 version there can be breaking changes.

Reply:
Alright, we'll keep it then.